lib/pipeline/configure.go

```diff
@@ -28,7 +28,7 @@ func NewConfigureResource(
 	logger logr.Logger,
 ) ([]client.Object, error) {
 
-	pipelineResources := NewPipelineArgs(promiseIdentifier, resourceRequestIdentifier, pipeline.Name, rr.GetName(), rr.GetNamespace())
+	pipelineResources := NewPipelineArgs(promiseIdentifier, rr.GetName(), pipeline.Name, rr.GetName(), rr.GetNamespace())
```

Comment:
`rr.GetName()` seems to be passed twice to the function 👀

Reply:
Good spot! Definitely not necessary